src/assests/styles/colors.css

```diff
@@ -0,0 +1,19 @@
+@import url('https://fonts.googleapis.com/css2?family=Gideon+Roman&display=swap');
+
+:root {
+    --primary: #F8F9FA;
+    --theme: #212529;
+    --light-grey: #CED4DA;
+    --little-dark-grey: #495057;
+    --little-light-grey: #ADB5BD;
+    --red-color:#EF233C;
+    --green-color: #2B9348;
+    --bottom-shadow: 0px 15px 10px -15px var(--light-grey);
+    --top-shadow: 0px -15px 10px -15px var(--light-grey);
+    --right-bottom-shadow: 15px -10px 15px -15px var(--little-dark-grey);
+    --all-side-shadow: 0px 0px 2px 2px var(--little-light-grey);
+    --all-side-bold-shadow: 0px 0px 10px 10px var(--little-dark-grey);
+    --font: 'Gideon Roman', cursive;
+    --all-side-red-border: 0px 0px 2px 2px var(--red-color);
+    --all-side-green-border: 0px 0px 2px 2px var(--green-color);
+}
```

Comment:
If you are already importing your component library, why are you defining extra colors here, they seem redundant?
Ignore if these are additional

Reply:
It is because the colors are different and shadows are not implemented in component library